src/Controller/ProductsController.php

```diff
@@ -33,6 +33,14 @@ public function products(): Response
         ]);
 
     }
+    #[Route('/product/{id}/delete', name: 'product_delete')]
+    public function productDelete($id): Response
+    {
+        $product = $this->productRepository->find($id);
+        $this->entityManager->remove($product);
+        $this->entityManager->flush();
+        return $this->redirectToRoute('products');
```

Comment:
```suggestion

        return $this->redirectToRoute('products');
```
Whitespace before return gives clearer readability